docs/framework/database/schema-design.md

```diff
@@ -13,10 +13,10 @@ Another method of viewing CiviCRM's Database model is to use one of various MySQ
 1. Look at the list of tables in the database. The table name generally gives you a good idea of the type of data it stores.
 2. Pick one table and look closer. Let's look at `civicrm_country` (a relatively simple example) by opening the "Table inspector" for that table.
     - We can look at the list of columns in the table.
-    ![MySQL Workbench CiviCRM Country Columns](/img/mysql_workbench_civicrm_country_tables.png)
+    ![MySQL Workbench CiviCRM Country Columns](/docs/img/mysql_workbench_civicrm_country_tables.png)
```

Comment:
Your links does work properly. I was reading the docs from github (https://github.com/civicrm/civicrm-dev-docs/blob/master/docs/framework/database/schema-design.md) and the links would give me 404 but now I just realized that this repo generates the docs for docs.civicrm.org... 

You can discard this pull request, I'm sorry for taking your time.